modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `inline_boundaries` feature now supports the `home`, `end`, `pageup` and `pagedown` keys #TINY-4612
 - Added new `PAGE_UP` and `PAGE_DOWN` key code constants to the `VK` API
 - Added support for alpha list numbering to the `lists` plugin #TINY-6891
+- Added new buttons for newly created table actions #TINY-7163
+- Added new icons for the newly added table actions #TINY-7157
+- Added the new command `mceTableToggleColumnHeader` which toggles currently selected column to or from a header #TINY-7163
```

Comment:
When we were doing the table work in TinyMCE 5.5, we specifically said we did **not** want toggle commands. I'm trying to remember why, but I believe it was because many have more than 2 states and as such it didn't make sense to include them as they'd be confusing.

@metricjs can you recall anything else about why we opted not to provide toggle commands for header types?